### PR TITLE
Interactive

### PR DESCRIPTION
**Pull request by shadabwahidullah**
## I have added the following features to my todo list:
- Adding a new task
- editing previous tasks
- checking completed tasks
- deleting a task
- deleting all completed tasks

## note: When you are done editing a task you have to press enter to save the changes

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,21 +12,22 @@
...
         </div>
-        <div class="flex-space-between">
+        <form class="newTask flex-space-between">
           <input
-            class="text"
+            class="newTaskName text"
             placeholder="Add to your list"
             type="text"
             name=""
             id="newTask"
```
Comment (rloterh) [inline src/index.html:26]: - [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Index mismatch in your application:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Wahidullah, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍 
- CRUD feature is functioning as intended
- Functions effectively split into various files
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (shadabwahidullah) [reply src/index.html:26]: thanks for mentioning that.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @shadabwahidullah, 

### Status: Approved ✔️

:+1:  You have implemented all the requested changes. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- I would recommend removing the unnecessary files from Github. Ex: `.vscode`. It will make the project structure bad. Please add this folder to `.gitignore`.


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**